params/btc/btc_genesis.go

```diff
@@ -83,7 +84,7 @@ var genesisBlock = types.Block{
 		TxRoot:     genesisMerkleRoot,        // 4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b
 		Timestamp:  time.Unix(0x495fab29, 0), // 2009-01-03 18:15:05 +0000 UTC
 		Difficulty: 0x1d00ffff,               // 486604799 [00000000ffff0000000000000000000000000000000000000000000000000000]
-		Nonce:      0x7c2bac1d,               // 2083236893
+		Pow:      &pow.Blake2bd{},               //
```

Comment:
The btc params can't co-exist anyway, the `btc` folder should be removed with the pow refactoring.

Reply:
had remove the btc and dcr params dependent dependence